packages/edit-post/src/components/visual-editor/index.js

```diff
@@ -101,6 +141,19 @@ export default function VisualEditor( { styles } ) {
 
 	const blockSelectionClearerRef = useBlockSelectionClearer( true );
 
+	// Allow scrolling "through" popovers over the canvas. This is only called
+	// for as long as the pointer is over a popover. Do not use React events
+	// because it will bubble through portals.
+	const toolWrapperRef = useRefEffect( ( node ) => {
```

Comment:
If feels like this has the potential to break popovers with scrollbars though?

Reply:
Those are rendered in the normal popover slot, which is outside this div. This slot is solely used for popovers that are placed over the content.